modules/mod_ginger_base/models/m_ginger_rest.erl

```diff
@@ -64,11 +64,14 @@ with_edges(Rsc = #{<<"id">> := Id}, Predicates, Context) ->
     Rsc#{<<"edges">> => Edges}.
 
 %% @doc Get resource translations.
--spec translations(atom() | {trans, proplists:proplist()}, z:context()) -> translations().
+-spec translations(binary() | {trans, proplists:proplist()}, z:context()) -> translations().
 translations({trans, Translations}, Context) ->
-    [{Key, z_html:unescape(filter_show_media:show_media(Value, Context))} || {Key, Value} <- Translations];
+    lists:foldl(fun({Key, Value}, Acc) -> 
+		       V = z_html:unescape(filter_show_media:show_media(Value, Context)),
+		       Acc#{Key => V}
+	       end, #{}, Translations);
 translations(Value, Context) ->
-    [{z_trans:default_language(Context), Value}].
+    #{z_trans:default_language(Context) => Value}.
```

Comment:
👍 but this looks unrelated to the scope of this PR. After all, in the coordinates search result no translations are returned (and therefore encoded/decoded) at all.